### DISCUSSION TITLE
Set custom label for ContentType in admin menu.

### DISCUSSION THREAD

**Discussion by yammycode** [bolt/core#2089] [Ideas / Feature requests]
Please add this feature! Suppose I created my ContentType "news". The name of this item should also be News. But I want this item to appear on the menu as "Новости", in Russian. Or some other title. As far as I understand, there is no such possibility now.

In the development of the topic, it would be great if it would be possible to specify a label with reference to the user's language. For example, if the language 'ru' is selected - Новости, if 'en' or default- News. And if the label is not specified, then the Name is displayed.

The same applies to other fields. The system has a very cool implementation of internationalization, but without the ability to specify values ​​for different languages, its meaning is small ...

**Comment by I-Valchev** [comment]
hey @yammycode, that's a great idea 👍! In fact, all we have to do to make the backend itself localized, is make sure to use more Symfony labels: so that when people add it, they can also translate the things into different locales.

What I do not understand, is why would it be different than the `name` and/or `singular_name` of the ContentType? Can you explain, maybe it is a use-case that I'm failing to think of?

**Comment by yammycode** [comment]
Sorry, I closed it by accident. Personally, I was faced with a situation when it is necessary to display menu items in Russian, so the menu item must differ from the name in English. But that's not all. I have a customer who loves to bring the admin panel to perfection. And renaming menu items is a common thing several times a day! It will not be easy to explain to such a customer that one of the best СMS does not have such an opportunity ...
